Fe design modifications

jhubbard2017 opened the pull request.

Commit:
Modified front end ui design for authentication components

Added border radius style to textfield inputs
This gives a more user friendly design for use interaction

Commit:
Modified profile components front end design

Profile component was updated to be cleaner and lighter
Edit profile component's widget box was removed and
text fields borders were rounded some.
Styles can now be added in a central location to increase usability
	and imported when needed

lmartin73 commented on src/app/components/styles/styles.js:
```diff
@@ -0,0 +1,20 @@
+export const textfieldStyle = {
```
is is necessary to define this?

jhubbard2017 replied:
I would think so, since its gonna be access by many inputs

lmartin73 replied:
ok

lmartin73 commented on src/app/routes/auth/containers/Forgot.js:
```diff
@@ -6,16 +6,20 @@ import DisplayContent from '../components/DisplayContent'
 const mapStateToProps = (state) => {
-    return {}
+    return {
+        state: state
```
do you really need to define the whole store to the state?

lmartin73 commented on src/app/routes/profile/components/profile.js:
```diff
@@ -1,75 +1,38 @@
+            <div className="col-lg-7 col-md-6 text-center" style={boxShadowStyle}>
+                <br/>
+                <img src="assets/img/avatars/sunny.png" style={bigCirclePhotoStyle} />
+                <h1 className="text-danger"><br/>{props.user.firstname + " "}<span className="semi-bold">{props.user.lastname}</span>
```
perhaps we should just map the last name to and so forth to it's own prop. that way this presentational view doesn't have to know about the user structure in the store.

lmartin73 commented on src/app/routes/profile/containers/editprofile.js:
```diff
@@ -219,147 +220,129 @@ class EditProfile extends React.Component {
+                                        <div className="col-lg-4 col-md-4">
+                                            <div className="text-center">
+                                                <div className="">
+                                                    <img className="img-thumbnail" name="image" ref="image" style={bigCirclePhotoStyle} src={this.state.photoURL}/><br/>
```
should this be 'this.profile.photoURL' as well as for the following places where you try to access the profile data from the local state?

Commit:
Fixed pull request changes

- Removed returning state from forgot password component
- Fixed profile component to include photoURL prop

Commit:
Updated commit from requested changes

Commit:
Removed mapStatesToProps from forgot password component
